src/components/ScrollDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import { AnimatePresence } from 'framer-motion'
+import type { PropsWithChildren } from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { FadeTransition } from './FadeTransition'
+
+type ScrollDisplayProps = {
+  threshold?: number // Threshold for when to switch displays, default to 72px
+} & PropsWithChildren
+
+type ScrollDisplayCompoundProps = {
+  Default: React.FC<{ children: React.ReactNode }>
+  OutOfView: React.FC<{ children: React.ReactNode }>
+}
+
+export const ScrollDisplay: React.FC<ScrollDisplayProps> & ScrollDisplayCompoundProps = ({
+  threshold = 72,
```

Comment:
We should move this guy to a constant to avoid the magic number.

Reply:
@0xApotheosis is there a constant file I can use, or should I put one somewhere else?

Reply:
Same file is fine for single use 🤝